modules/admin_manual/pages/enterprise/collaboration/msoffice-wopi-integration.adoc

```diff
@@ -22,6 +22,11 @@ Please bear in mind:
 
 TIP: If you want to integrate the {office365-url}[Office 365 (cloud)] version of Microsoft Office Online, you need to {oc-contact-url}[get in touch with us].
 
+== Preparing the Environment
+You need an https://docs.microsoft.com/de-de/officeonlineserver/deploy-office-online-server[OfficeOnline Server] installed.
```

Comment:
@voroyam info: please create the link in the form as it is made in the TIP above = definition on top and reference usage in the line. We have this procedure in all documents for better readability and managability.

Reply:
done